compiler/ballerina-parser/src/test/java/io/ballerinalang/compiler/parser/test/syntax/declarations/ObjectTypeDefinitionTest.java

```diff
@@ -79,72 +79,72 @@ public void testClientQualifierOnly() {
         test("object-type-def/object_type_def_source_14.bal", "object-type-def/object_type_def_assert_14.json");
     }
 
-    @Test
+    @Test(enabled = false) // no longer valid, due to spec deviation
```

Comment:
Lets remove these invalid tests. May be we can keep one or two as negative test cases.